scipy/integrate/_ivp/radau.py

```diff
@@ -245,6 +254,10 @@ class Radau(OdeSolver):
         is assumed to be dense.
     vectorized : bool, optional
         Whether `fun` is implemented in a vectorized fashion. Default is False.
+    mass : {None, array_like, sparse_matrix}, optional
```

Comment:
Should this and `mass_matrix` above have the same name?

Reply:
Indeed, this may be more clear. I have updated this.